src/System.Runtime.Loader/tests/System.Runtime.Loader.Noop.Assembly/project.json

```diff
@@ -1,17 +1,9 @@
 {
   "dependencies": {
+    "Microsoft.NETCore.Platforms": "1.0.1-rc3-24102-00",
     "System.Runtime": "4.1.0-rc3-24102-00"
   },
   "frameworks": {
     "netstandard1.0": {}
-  },
-  "runtimes": {
-    "win7-x86": {},
-    "win7-x64": {},
-    "ubuntu.14.04-x64": {},
-    "osx.10.10-x64": {},
-    "centos.7-x64": {},
-    "rhel.7-x64": {},
-    "debian.8.2-x64": {}
```

Comment:
Huh, well, if everything still works like this, then you should be able to simplify and remove the `Platforms` package since there are no runtimes. (Same for `System.Diagnostics.FileVersionInfo.TestAssembly` and `System.Runtime.Loader.Test.Assembly`.)